packages/launchdarkly-adapter/modules/adapter/adapter.spec.js

```diff
@@ -1,3 +1,4 @@
+import { TAdapterConfigurationStatus } from '@flopflip/types';
```

Comment:
Nit: this is supposed to be an enum, it's a bit weird to name is as a type. I'm not a naming expert but maybe we can make it less ambiguous.

For example: `AdapterConfigurationStatusEnum`